store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -2473,9 +2459,14 @@ public void call(String value) {
     }
 
     public void deleteMailbox(DeleteBlobs deleteBlobs) throws ServiceException {
-        StoreManager sm = StoreManager.getInstance();
+        final String genericStoreManagerClass = "com.zimbra.storemanagers.store.GenericStoreManager";
```

Comment:
why is genericStoreManagerClass being defaulted to "com.zimbra.storemanagers.store.GenericStoreManager"?